data/campaigns/tutorial03_seafaring.wmf/scripting/texts.lua

```diff
@@ -176,3 +176,88 @@ conclusion = {
    ),
    h = 450
 }
+
```

Comment:
as this is started after the conclusion it sounds weird. we need to cut the part about tutorial done play further if you want from the previous conclusion text and move it to the end of the tutorial